src/main/kotlin/no/nav/syfo/Bootstrap.kt

```diff
@@ -142,57 +145,61 @@ suspend fun blockingApplicationLogic(
-                keyValue("orgNr", "missing"),
-                keyValue("msgId", "missing"),
-                keyValue("sykmeldingId", "missing")
-        )
+        kafkaConsumer.poll(Duration.ofMillis(0)).forEach {
+            val produceTask = it.value().produceTask
+            val registerJournal = it.value().registerJournal
 
-        val logKeys = logValues.joinToString(prefix = "(", postfix = ")", separator = ",") {
-            "{}"
-        }
+            val loggingMeta = LoggingMeta(
+                    orgNr = produceTask.orgnr,
+                    msgId = registerJournal.messageId,
+                    sykmeldingId = it.key()
+            )
 
-        kafkaConsumer.poll(Duration.ofMillis(0)).forEach {
-                val produceTask = it.value().produceTask
-                val registerJournal = it.value().registerJournal
-                logValues = arrayOf(
-                        keyValue("sykmeldingId", it.key()),
-                        keyValue("orgNr", produceTask.orgnr),
-                        keyValue("msgId", registerJournal.messageId)
-                )
-                log.info("Received a SM2013, going to create oppgave, $logKeys", *logValues)
-                val opprettOppgave = OpprettOppgave(
-                        tildeltEnhetsnr = produceTask.tildeltEnhetsnr,
-                        aktoerId = produceTask.aktoerId,
-                        opprettetAvEnhetsnr = produceTask.opprettetAvEnhetsnr,
-                        journalpostId = registerJournal.journalpostId,
-                        behandlesAvApplikasjon = produceTask.behandlesAvApplikasjon,
-                        saksreferanse = registerJournal.sakId,
-                        beskrivelse = produceTask.beskrivelse,
-                        tema = produceTask.tema,
-                        oppgavetype = produceTask.oppgavetype,
-                        aktivDato = LocalDate.parse(produceTask.aktivDato, DateTimeFormatter.ISO_DATE),
-                        fristFerdigstillelse = LocalDate.parse(produceTask.fristFerdigstillelse, DateTimeFormatter.ISO_DATE),
-                        prioritet = produceTask.prioritet.name
-                )
-
-                val response = oppgaveClient.createOppgave(opprettOppgave, registerJournal.messageId)
-                OPPRETT_OPPGAVE_COUNTER.inc()
-                log.info("Task created with {} $logKeys",
-                        keyValue("oppgaveId", response.id),
-                        keyValue("sakid", registerJournal.sakId),
-                        keyValue("journalpost", registerJournal.journalpostId),
-                        keyValue("tildeltEnhetsnr", produceTask.tildeltEnhetsnr),
-                        *logValues)
+            wrapExceptions(loggingMeta) {
+                handleRegisterOppgaveRequest(oppgaveClient, produceTask, registerJournal, loggingMeta)
+            }
         }
         delay(100)
     }
 }
 
+@KtorExperimentalAPI
+suspend fun handleRegisterOppgaveRequest(
```

Comment:
Har du vurdert å dra dette ut i en egen fil? Kanskje under egen pakke for domenet oppgavehåndtering?